lib/pluginmanager/update.rb

```diff
@@ -8,25 +8,28 @@ class LogStash::PluginManager::Update < LogStash::PluginManager::Command
   def execute
     local_gems = gemfile.locally_installed_gems
 
-    if update_all? && !local_gems.empty?
-      error_plugin_that_use_path!(local_gems)
-    else
-      plugins_with_path = plugins_arg & local_gems
-      error_plugin_that_use_path!(plugins_with_path) if plugins_with_path.size > 0
+    if local_gems.size > 0
```

Comment:
I usually go with positive conditionals, this is why I've changed it to check for the size instead of using `empty?`
if we had active_support I could go with `local_gems.present?`, ahh rails ;)